src/Model/Table/SentencesListsTable.php

```diff
@@ -82,7 +82,7 @@ private function _getPermissions($list, $currentUserId) {
             'canView' => $visibility != 'private' || $belongsToUser,
             'canEdit' => $belongsToUser,
             'canAddSentences' => $belongsToUser && $editableBy !== 'no_one',
-            'canRemoveSentences' => $belongsToUser || $editableBy == 'anyone',
+            'canRemoveSentences' => $editableBy == 'anyone' || $belongsToUser && $editableBy == 'creator',
```

Comment:
The new value for `canRemoveSentences` is definitely correct now. Now I realize that the value for `canAddSentences` is incorrect. So the "Add a sentence to this list" form on the list page doesn’t show up for other users when the list is editable by anyone. Feel free to submit a new PR that fixes that.

Reply:
I'll do that later, yes.